CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 
 ### Changed
 
+- Added  `.hashtag`, .`mention` to detect theses pattern inside the `messageLabel`.
+- Added `.custom(pattern: YOUR_PATTERN)` to `DetectorType` to manage and deal with your own regular expression.
+[#913](https://github.com/MessageKit/MessageKit/pull/913) by [@JulienKode](https://github.com/julienkode).
+
```

Comment:
There should only be one entry for this. It should also go under the `Added` section under `Upcoming Release`

Reply:
@SD10 Thanks I've just fix it

Reply:
Sorry but two more things:
- this is also an API breaking change. Can you mark it as **Breaking Change** like the other entries?
- this needs an `### Added` section above the entry

Reply:
@SD10 Ok I've just added it